Client/src/main/java/com/lnc/controller/LoginController.java

```diff
@@ -0,0 +1,91 @@
+package com.lnc.controller;
+
+import com.lnc.service.AuthService;
+import com.lnc.util.InputHandler;
+
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+public class LoginController {
+  AuthService auth = new AuthService();
+
+  public void loginUser() throws Exception {
+    String employeeID = getEmployeeID();
+    String password = getPassword();
+    String hashPassword = getHashPassword(password);
+
+    auth.authenticate(employeeID, hashPassword);
+  }
+
+  private String getEmployeeID() throws IOException {
```

Comment:
This method violating SRP, it is saying it will getEmployeeId but it's 
1. Asking for customer input
2. Validating
3. Also printing